content/llm/speaking-bots.md

```
@@ -985,6 +985,19 @@ Each endpoint is documented with:
 Use the navigation to explore the available endpoints.
 
 
+---
+
+## Generate Persona Image
+
+### Source: ./content/docs/speaking-bots/reference/personas/generate_persona_image_personas_generate_image_post.mdx
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)